Project.toml

```diff
@@ -0,0 +1,11 @@
+name = "NIfTI"
+uuid = "f897abbf-c5d9-45cb-bf89-9f7f579a29ec"
+authors = ["Petr Krysl <pkrysl@ucsd.edu>"]
```

Comment:
Could we delete this line? I don't know if we should or shouldn't have distinct authorship for this package. That should probably be addressed at another time.

Reply:
Absolutely no problem. It was generated automatically.